package.json

```diff
@@ -207,6 +214,9 @@
         "visible": true,
         "subPermissions": [
           "configuration.all",
+          "orders.configuration.prefixes.all",
+          "orders.configuration.reasons-for-closure.all",
```

Comment:
what's about closing orders? IMHO, the best way to test permissions - to create a new user, Assigned some order permissions and trying to go through different workflows with him logged in